releng/org.eclipse.nebula.nebula-parent/pom.xml

```diff
@@ -22,7 +22,7 @@ Contributors:
 
 	<properties>
 
-		<tycho-version>2.6.0</tycho-version>
+		<tycho-version>3.0.4</tycho-version>

```

Comment:
You might want to use a `maven.config` file and then share the Tycho version for extension/pom in that file like described here: https://tycho.eclipseprojects.io/doc/latest/StructuredBuild.html#pomless-builds